src/libsync/discovery.cpp

```diff
@@ -329,6 +329,13 @@ void ProcessDirectoryJob::processFile(PathTuple path,
                               << " | e2ee: " << dbEntry._isE2eEncrypted << "/" << serverEntry.isE2eEncrypted
                               << " | e2eeMangledName: " << dbEntry.e2eMangledName() << "/" << serverEntry.e2eMangledName;
 
+    if (localEntry.isValid()
+        && serverEntry.isValid()
```

Comment:
@FlexW Shouldn't there be a `!` before `serverEntry.isValid()`? According to a warning that's being logged, the file is **not** in the sync journal and **server**.

Reply:
Yes, of course. I thought I did it. Thanks for the catch.